rqt_image_view/src/rqt_image_view/image_view.cpp

```diff
@@ -280,6 +292,49 @@ void ImageView::saveImage()
   img.save(file_name);
 }
 
+void ImageView::onMousePublish(bool checked)
+{
+  if(checked)
+  {
+    std::string topicName;
+    if(pub_topic_custom_)
+    {
+      topicName = ui_.publish_click_location_topic_line_edit->text().toStdString();
+    } else {
+      if(!subscriber_.getTopic().empty())
+      {
+        topicName = subscriber_.getTopic()+"_mouse_left";
+      } else {
+        topicName = "mouse_left";
```

Comment:
Wouldn't it be easier (and more intuitive to the user) if the line edit widget just has this value as a default? Then the user also seems the default.